circle.yml

```diff
@@ -3,23 +3,23 @@ version: 2
 jobs:
   swift5:
     docker:
-      - image: swift:5.0
+      - image: swift:5.2
```

Comment:
I made the absolute minimal changes here, happy to change to whatever you suggest though.